<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="tileScrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <widget class="TileDisplay" name="tileDisplay">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>627</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QWidget" name="sideBar" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="pageLabel">
         <property name="text">
          <string>Map16 Page:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pageSelect">
         <property name="prefix">
          <string>0x</string>
         </property>
         <property name="maximum">
          <number>166</number>
         </property>
         <property name="displayIntegerBase">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tilesetLabel">
         <property name="text">
          <string>Tileset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tilesetSelect"/>
       </item>
       <item>
        <widget class="QLabel" name="paletteLabel">
         <property name="text">
          <string>Palette:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="paletteSelect"/>
       </item>
       <item>
        <widget class="QLabel" name="scaleLabel">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="scaleSelect">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="tileLabel">
         <property name="text">
          <string>Tile:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tileText">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TileDisplay</class>
   <extends>QWidget</extends>
   <header>tile_display.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
